atc/metric/emit.go

```diff
@@ -110,6 +116,9 @@ func Deinitialize(logger lager.Logger) {
 }
 
 func emit(logger lager.Logger, event Event) {
+	logger.Debug("emit-event", lager.Data{
```

Comment:
We already have a way of having the metrics emitted logging to `stdout` through the `--emit-to-logs` variable - did you want this logline for the purposes of checking if metrics are still being emitted regardless of the configured emitter, or for the same purposes as `--emit-to-logs`?

Reply:
This code is to check regardless of the configured emitter. If I remember correctly only one of the emitters can be configured at a time. So we cannot --emit-to-logs and InfluxDB emission configured at the same time.

Reply:
I'm lukewarm on this; it seems like it'll be super noisy but I guess you can always filter it out. It's already at debug level but we have a decent amount of environments with debug logs enabled. Granted, we're used to maintaining filters to strip out the cruft.

If it was just added while you were developing and isn't useful anymore I would prefer we remove it, but if it was super useful in a pinch I'm fine with keeping it. :man_shrugging: